packages/react-composites/src/composites/CallComposite/components/ComplianceBanner.tsx

```diff
@@ -12,27 +12,47 @@ export type ComplianceBannerProps = {
   callRecordState?: boolean;
 };
 
+type CachedComplianceBannerProps = {
+  new: ComplianceBannerProps;
```

Comment:
nitty nit: just a personal taste on not using 'new' as an only word.
So, maybe 'coming'/'next'/'current' instead, and 'previous' instead of 'old'